site/docs/pages/index.mdx

```diff
@@ -288,173 +386,154 @@ setFilteredTokens(filteredTokens);
       </div>
     </div>
   </aside>
-</section>
-
-:::danger
-**Note: This interface is for demonstration purposes only.**
-
-The swap will execute and work out of the box when you implement the component in your own app.
-:::
-
-{' '}
 
-<hr className="my-8 md:my-20" />
-
-<section>
-  <main className="flex flex-row gap-1">
-    <h2 className="basis-1/2 text-4xl md:text-6xl">Utilities</h2>
+  <main id="frame" className="pt-[56px] flex flex-col gap-4 md:flex-row md:text-left text-center">
+    <h3 className="basis-1/2 py-[15px] text-4xl md:text-4xl">Frame</h3>
   </main>
-  <p className="py-2 text-lg md:text-xl">
-    If you're seeking basic TypeScript utilities, we have plenty of ready-to-use
```

Comment:
@abcrane123  Oh! ops, this should still be here.

Please add it back in the home page :D 

it's important to showcase all our utilities, especially for SEO reasons.